src/java.base/share/classes/java/util/concurrent/AbstractExecutorService.java

```diff
@@ -261,9 +261,8 @@ public <T> T invokeAny(Collection<? extends Callable<T>> tasks,
      */
     @Override
     public <T> List<Future<T>> invokeAll(Collection<? extends Callable<T>> tasks)
-        throws InterruptedException {
-        if (tasks == null)
-            throw new NullPointerException();
+        throws InterruptedException , NullPointerException {
```

Comment:
NullPointerException is a RuntimeException

```suggestion
        throws InterruptedException {
```